config.js.example

```diff
@@ -1,11 +1,11 @@
 /* Security */
-var ezid_username = "apitest";
-var ezid_password = "apitest";
+var ezid_username = "sml.tcia";
+var ezid_password = "eguk-lude-vich";
```

Comment:
are these supposed to be on github? @birm 

PS: sorry still get notifications for these repositories, going to mute!

Reply:
Thanks for catching 💯 🥇